lib/src/sql/value/value.rs

```diff
@@ -886,6 +886,7 @@ impl Value {
 				Function::Script(_, _) => "fn::script".to_string().into(),
 				Function::Normal(f, _) => f.to_string().into(),
 				Function::Cast(_, v) => v.to_idiom(),
+				Function::Not(v) => v.to_idiom(),
```

Comment:
I don't fully understand the utility of converting functions into idioms, and therefore cannot affirm the correctness of this line.